content/W2015/W15-event-confidence.md

```diff
@@ -0,0 +1,25 @@
+Title: Big CSters Discussion Circle: Self-Confidence
+Date: 2015-03-15 20:00
+Category: Events
+Tags: social, big csters
+Slug: csters-confidence
+Author: Julia Nguyen
+Summary: Big CSters convenes for a discussion circle around the topics of imposter syndrome and self-confidence.
+
+This week's discussion circle theme is self-confidence. We've all faced 
+imposter syndrome&mdash;feeling like we don't deserve the success we've 
+achieved.  Inspired by the Write/Speak/Code workshops, we'll be practicing 
+strategies to own up to our achievements and improve self-confidence. Like the 
+previous event, we'll also be serving a delicious catered meal! Come out and 
+meet your fellow CSters!
```

Comment:
yeah, we decided catering after the blurb was written. Could you change to 
"we'll also be serving a delicious catered meal - this week it's Zoup! soups and sandwiches" or something along those lines?